conf/VERSION.properties

```diff
@@ -1,4 +1,4 @@
 
 ## Defines the JGroups version
 ## This file should be the *only place* to change when changing the version number
-jgroups.version=3.6.4.Final
\ No newline at end of file
+jgroups.version=4.0.0-SNAPSHOT
```

Comment:
Could you please elaborate more why, and what kind of fix are you suggesting here? Unfortunately in JBoss versioning rules I couldn't find any paragraph dedicated to snapshots...

Looking at some examples [1][2][3] the proposed convention (`4.0.0-SNAPSHOT`) looks absolutely fine.

[1] https://repository.jboss.org/nexus/service/local/repositories/snapshots/content/org/hibernate/hibernate-c3p0/3.6.10-SNAPSHOT/maven-metadata.xml
[2] https://repository.jboss.org/nexus/service/local/repositories/snapshots/content/org/infinispan/infinispan-bom/7.2.2-SNAPSHOT/maven-metadata.xml
[3] https://repository.jboss.org/nexus/service/local/repositories/snapshots/content/org/drools/drools-bom/6.3.0-SNAPSHOT/maven-metadata.xml